_posts/2018-11-25-Rueckblick-auf-die-Breminale-2018

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  Rückblick auf die Breminale 2018
+author: Louis
+date:   2018-11-25 00:19:00 +0200
+---
+# Rückblick auf die Breminale 2018
+Wir haben es wieder vom Osterdeich in unser Freifunk-Hauptquartier in den [Hackerspace](https://www.hackerspace-bremen.de)
+geschafft. Nach längerer Funkstille bezüglich der Breminale wollen wir noch einen kleinen Rückblick auf diese phänomenalen
+Tage werfen.
+
+## Die Vorbereitung
+Am Mittwoch ging die Breminale los. Wir standen ab Samstag davor am Deich. Zugegeben, wirklich gestanden hat keiner von uns.
+Die Software-Fraktion unter uns hat hauptsächlich gesessen, die Hardware-Fraktion hat Kabel und Geräte von A nach B
+getragen - und ist damit hauptsächlich gelaufen.
+Von TP-LINK haben wir in diesem Jahr 80 Router des Typs Archer C7 gestellt bekommen. Uns wurde auch angeboten,
+Professionellere Geräte zu nutzen. Wir haben uns dagegen entschieden gehabt, denn bei den nun verwendeten Geräten
+handelt es sich um Router, die langfristig auch im gewöhnlichen Freifunk-Netz genutzt werden können.
+In diesem Jahr haben wir auf den Routern auf der Breminale unsere reguläre Firmware mit ein paar Anpassungen verwendet.
+Die muss aber erst mal auf 80 Geräte drauf. Mit geballter Menpower haben wir eine Flash-Party veranstaltet. Falls beim
+ein oder anderen jetzt das Lied von Queen im Kopf aufploppt: Uns ging es [ähnlich](https://flash.timi.one).
+Dieses Jahr hatten wir bis auf einen starken, kurzen Regenschauer keine Probleme mit Nässe von oben. Und durch günstige
+Tide-Zeiten auch keine Probleme mit Nässe von unten. Dafür hat die Sonne in diesem Jahr ihr Bestes gegeben. Bereits ab
+Tag zwei des Aufbaus wurden die Helfer unbekleideter, ab Tag drei waren T-Shirts die Ausnahme. Ab Tag vier musste man
+Aufpassen, dass der Schweiß nicht auf die Router tropft. 
+
+Neu in diesem Jahr war ein übergroß (länger als 4 m) ausgedruckter Veranstaltungsplan. Analog for the win. Auf ihm
+konnten wir in anständiger Große die Positionen unserer Hardware markieren. Das hat gut geholfen, um auch Personen
+einen Überblick über das Netz zu geben, die nicht von Anfang an dabei waren.
+<a href="/blog/files/2018-11-25/breminale_plan.jpg"><img src="/blog/files/2018-11-25/breminale_plan.jpg" alt="Plan mit Menschen, die darauf Zeigen" style="max-height:400px"></a>
+
+## Der Versuch
+Neben dem Anspruch, tausende Besucher mit Internet per WLAN zu versorgen ist die Breminale für uns auch eine Spielwiese.
+Wie reagiert unsere Firmware mit einem großen Mesh-Netz? Wie kommen die Router mit massig Clients klar? Wir haben
+außerdem versucht, auf eine neue Technologie zu setzen, die die Geschwindigkeit im Freifunk-Netz rapide erhöhen kann:
+Babel. Leider mussten wir am Dienstag feststellen, dass sie noch an ein ein paar Stellen ausbesserungswürdig ist.
+Unter anderem hat die Umsetzung von IPv4- auf IPv6-Adressen und zurück nicht zuverlässig funktioniert. Somit ist
+die bisherige Software-Technik verwendet worden.
```

Comment:
Nur leicht modifiziert worden, sodass Clients mit 5 GHz, das 2.4 GHz Band nicht angeboten wurde. 

_Villt. sowas in der Richtung für einfache Leser_